cvat-ui/src/components/change-password-modal/change-password-form.tsx

```diff
@@ -21,6 +21,40 @@ interface Props {
     onSubmit(loginData: ChangePasswordData): void;
 }
 
+export const validatePassword: RuleRender = (): RuleObject => ({
+    validator(_: RuleObject, value: string): Promise<void> {
+        if (!patterns.validatePasswordLength.pattern.test(value)) {
+            return Promise.reject(new Error(patterns.validatePasswordLength.message));
+        }
+
+        if (!patterns.passwordContainsNumericCharacters.pattern.test(value)) {
+            return Promise.reject(new Error(patterns.passwordContainsNumericCharacters.message));
+        }
+
+        if (!patterns.passwordContainsUpperCaseCharacter.pattern.test(value)) {
+            return Promise.reject(new Error(patterns.passwordContainsUpperCaseCharacter.message));
+        }
+
+        if (!patterns.passwordContainsLowerCaseCharacter.pattern.test(value)) {
+            return Promise.reject(new Error(patterns.passwordContainsLowerCaseCharacter.message));
+        }
+
+        return Promise.resolve();
+    },
+});
+
+export const validateConfirmation: (firstFieldName: string) => RuleRender = (firstFieldName: string): RuleRender => ({
```

Comment:
you have 2 arguments `firstFieldName`. Are they equal? If yes - why do we need the second